counterparty-core/counterpartycore/lib/sentry.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+import sentry_sdk
+
+from counterpartycore.lib import config, database
+from counterpartycore.lib.telemetry.collector import TelemetryCollectorLive
+
+
+def send_exception(exception):
+    sentry_sdk.capture_exception(exception)
+
+
+environment = os.environ.get("SENTRY_ENVIRONMENT", "development")
+
+release = os.environ.get("SENTRY_RELEASE", config.__version__)
+
+
+def before_send(event, _hint):
+    db = database.get_connection(read_only=True)
+
+    data = TelemetryCollectorLive(db).collect()
+
+    event["tags"] = event.get("tags", {})
+
+    event["tags"]["core_version"] = data["version"]
+    event["tags"]["addrindexrs_version"] = data["addrindexrs_version"]
+    event["tags"]["docker"] = data["docker"]
+    event["tags"]["network"] = data["network"]
+    event["tags"]["force_enabled"] = data["force_enabled"]
+
+    event["extra"] = event.get("extra", {})
+    event["extra"]["last_block"] = data["last_block"]
+
+    return event
+
+
+sentry_sdk.init(
```

Comment:
Can we record the hostname for Sentry? (but *not* for the node telemetry, which is on by default)

Reply:
Sentry automatically picks up hostname so I think we currently have the behavior you want.

Reply:
ah hm, I'm seeing `API-SERVER-{H,J,5,6}` etc.

Reply:
That is the error ID.  API-SERVER is the name of the project and the alphanumeric chars are the unique ID for a given error class ( as opposed to instance ).  We could rename API-SERVER to whatever we want. the hostname ( i.e. api3 or api4 ) is captured under `server_name` tag.